src/content/blog/astro-440.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "Astro 4.4"
+description: "Astro 4.4 is now available! This release includes the addition of performance audits for the dev toolbar, performance upgrades, the ability to automatically infer the dimensions of remote images, and more."
+publishDate: "February 15, 2024"
+authors:
+  - erika
+  - ema
+  - matthew
+  - nate
+  - bjorn
+coverImage: "/src/content/blog/_images/astro-440/post-header-4.4.webp"
+socialImage: "/src/content/blog/_images/astro-440/og-image-4.4.webp"
+lang: "en"
+---
+
+import perfAuditImage from "./_images/astro-440/perf-audit.webp"
+import BlogContentImage from "/src/components/BlogContentImage.astro"
+
+Astro 4.4 is now available! This release includes performance audits for the dev toolbar, performance upgrades, the ability to automatically infer the dimensions of remote images, and more.
+
+Highlights include:
+
+- [**Performance Audits**](#performance-audits)
+- [**Audit list**](#audit-list)
+- [**Improved streaming performance**](#improved-streaming-performance)
+- [**New `inferSize` properties for remote images**](#automatically-infer-dimensions-of-remote-images)
+
+
+## How to upgrade
+
+To take advantage of the latest features, make sure you're running the latest version of Astro. You can upgrade to Astro 4.4 by running the `@astrojs/upgrade` command:
+
+```sh
+npx @astrojs/upgrade
+```
+
+or by running the upgrade command for your package manager:
+
+```sh
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Performance audits
+
+Astro 4.4 includes the addition of performance audits for the dev toolbar. Much like the accessibility audits currently available, performance audits will help you identify and fix performance issues in your Astro site. For example, the dev toolbar will now warn you when a lazy-loaded image is above the fold, recommending instead that you use eager loading for better performance.
```

Comment:
What does `above the fold` mean?

Reply:
It's a common term: https://en.wikipedia.org/wiki/Above_the_fold

It means anything that doesn't require scrolling. I don't really think we need to define it in the article itself